codes/classical/properties/locally_recoverable.yml

```diff
@@ -0,0 +1,39 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+
+code_id: locally_recoverable
+
+
+name: 'Locally Recoverable code'
+
+description: 'A code \(C\) over \(\mathbb{F}_q^{n}\) with length \(n\) is \(r\)-locally recoverable, or have locality \(r\), if \(\forall i \in [n]\), \(\exists I_i \subset [n]\setminus i\)~ s.t. \(|I_i|\leq r\) and the projection of the set \(\mathcal{C}(i,a)=\{x\in C : x_i=a\}\) on to the coordinates in \(I_i\), i.e., \(\mathcal{C}_{I_i}(i,a)\) is disjoint from \(\mathcal{C}_{I_i}(i,a^\prime)\) where \(a\neq a^\prime\). In other words, given a codeword \(x\) and coordinate \(i\), \(x_i\) can be recovered from at most \(r\) other coordinates of \(x\). An \(r\)-locally recoverable code with length \(n\) and dimension \(k\) is denoted as (n,k,r) LRC code.'
```

Comment:
The definition usually includes recovering more than one coordinate, say t coordinates, from a small number of other coordinates. Please update.

Reply:
i think the first sentence says for all i. i changed the second sentence from "In other words, given a codeword \(x\) and coordinate \(i\),..." to "In other words, given a codeword \(x\) and any coordinate \(i\),...".

is that what you meant? i checked your papers and they use all coordinates i instead of a set of coordinates with cardinality less than t?